resources/js/blocks/ApplePayButtonComponent.js

```diff
@@ -11,6 +11,11 @@ export const ApplePayButtonComponent = ({cart, extensions}) => {
         shop: {countryCode, currencyCode = 'EUR', totalLabel = ''},
         ajaxUrl,
     } = mollieApplePayBlockDataCart
+    const { useMemo } = wp.element;
+    const style = useMemo(() => ({
```

Comment:
I am not a React expert, but I think `useMemo` should be used for very expensive actions. If you decide to remove it, you can ignore my comments about importing useMemo from  `@wordpress/element`